settings.gradle

```diff
@@ -2,13 +2,13 @@ rootProject.name="rollbar-java-sdk"
 
 include ":rollbar-api",
         ":rollbar-java",
-        ":rollbar-android",
+//        ":rollbar-android",
         ":rollbar-web",
         ":rollbar-log4j2",
         ":rollbar-logback",
         ":examples:rollbar-java",
-        ":examples:rollbar-web",
-        ":examples:rollbar-android",
-        ":examples:rollbar-scala",
-        ":examples:rollbar-log4j2",
-        ":examples:rollbar-logback"
+//        ":examples:rollbar-web",
+//        ":examples:rollbar-android",
+//        ":examples:rollbar-scala",
+        ":examples:rollbar-log4j2"
```

Comment:
This was the only example I wanted to run, so I took out the rest of the stuff to speed up the gradle builds.